telegram/_message.py

```diff
@@ -673,6 +707,9 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["Message"]:
             data.get("video_chat_participants_invited"), bot
         )
         data["web_app_data"] = WebAppData.de_json(data.get("web_app_data"), bot)
+        data["forum_topic_created"] = ForumTopicCreated.de_json(
+            data.get("forum_topic_created"), bot
+        )
```

Comment:
we also need to do de_json of `ForumTopicClosed` and Reopened

Reply:
Done, thank you. I got confused by the classes being empty. But it may change.